mm2src/lp_swap/maker_swap.rs

```diff
@@ -1178,6 +1209,13 @@ impl MakerSwap {
                         .await
                 );
 
+                broadcast_transaction_message(
+                    &self.ctx,
+                    tx_helper_topic(self.maker_coin.ticker()),
+                    transaction.tx_hex(),
+                    &self.p2p_privkey,
+                );
```

Comment:
What do you think about adding methods like `MakerSwap::broadcast_maker_coin_tx` and `MakerSwap::broadcast_taker_coin_tx`? Or `MakerSwap::broadcast_tx_message`.
Then it could be called easier:
```rust
self.broadcast_tx_message(self.maker_coin.ticker(), transaction.tx_hex())
```

Reply:
Yes this is how it should be actually. But I see so many functions that aren't coupled with their structs. For example, broadcasting functions of swap defined in the `lp_swap`. If I implement `broadcast_transaction_message` into `MakerSwap` and rest of the functions will stay the same, I thought it will increase complexity in coding structure.